.github/labeler.yml

```diff
@@ -0,0 +1,101 @@
+"area: Modem":
+  - "drivers/modem/**/*"
+"area: PWM":
+  - "drivers/pwm/**/*"
+"area: Watchdog":
+  - "drivers/watchdog/**/*"
+"area: Sensors":
+  - "drivers/sensors/**/*"
+"area: ADC":
+  - "drivers/adc/**/*"
+"area: Counter":
+  - "drivers/counter/**/*"
+"area: Timer":
+  - "drivers/timer/**/*"
+"area: I2S":
+  - "drivers/i2s/**/*"
+"area: C Library":
+  - "lib/libc/**/*"
+"area: Device Tree":
+  - "dts/**/*"
+  - "**/*.dts"
+"area: I2C":
+  - "drivers/i2c/**/*"
+"area: SPI":
+  - "drivers/spi/**/*"
+"area: Boards":
+  - "boards/**/*"
+"area: POSIX":
+  - "lib/posix/**/*"
+"area: native port":
+  - "arch/posix/**/*"
+  - "soc/posix/**/*"
+  - "**/*native_posix*"
+"area: X86":
+  - "arch/x86/**/*"
+"area: ARM":
+  - "arch/arm/**/*"
+  - "include/arch/arm/**/*"
+"area: NIOS2":
+  - "arch/nios2/**/*"
+"area: Xtensa":
+  - "arch/xtensa/**/*"
+"area: RISCv32/64":
+  - "arch/risv/**/*"
+"area: ARC":
+  - "arch/arc/**/*"
+"area: Networking":
+  - "subsys/net/**/*"
+  - "samples/net/**/*"
+  - "tests/net/**/*"
+  - "include/net/**/*"
+  - "include/drivers/ieee802154/**/*"
+  - "drivers/ethernet/**/*"
+  - "drivers/ieee802154/**/*"
+  - "drivers/wifi/**/*"
+  - "drivers/ptp_clock/**/*"
+  - "drivers/net/**/*"
+"area: Logging":
+  - "subsys/logging/**/*"
+"area: Shell":
+  - "subsys/shell/**/*"
+"area: Console":
+  - "subsys/console/**/*"
+"area: Test Framework":
+  - "subsys/testsuite/**/*"
+"area: Settings":
+  - "subsys/settings/**/*"
+"area: File System":
+  - "subsys/fs/**/*"
+"area: Storage":
+  - "subsys/storage/**/*"
+"area: Bluetooth":
+  - "subsys/bluetooth/**/*"
+  - "**/*bluetooth*"
```

Comment:
I'd still add include/bluetooth include/drivers/bluetooth drivers/bluetooth samples/bluetooth and tests/bluetooth to this list.

Reply:
those will already be covered with the second rule...